packages/dashboard/src/components/popoverMenu/story-menu-generator.js

```diff
@@ -53,6 +53,7 @@ export const generateStoryMenu = ({ menuItems, story, isLocked }) => {
         extraProperties.openNewTab = true;
         break;
       case STORY_CONTEXT_MENU_ACTIONS.RENAME:
+      case STORY_CONTEXT_MENU_ACTIONS.DELETE:
```

Comment:
Removing this line fixes the [disconnection of the test suite](https://github.com/GoogleForCreators/web-stories-wp/runs/5975213604?check_suite_focus=true). Not sure why it's disconnecting after adding this line

Reply:
@BrittanyIRL found it - looks like the second story was locked, so [this test](https://github.com/GoogleForCreators/web-stories-wp/pull/11214/files#diff-b34fffa3e835843ea476d0e161da20311ac20477fa909f1df32a3a45f2fc7092R627) was failing because it was targeting the locked story